### PR TITLE
opt(pingcap/tidb): skip test plugins for feature branches

### PR DESCRIPTION
**Pull request by wuhuizuo**
changed for feature branches that based on trunk or release-7.1+ branches.

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: pipelines/pingcap/tidb/latest/ghpr_build.groovy [hunk 1]
```diff
@@ -100,6 +100,7 @@ pipeline {
             }
         }
         stage("Test plugin") {
+            when { not { expression { REFS.base_ref ==~ /^feature[\/_]/ } } // skip for feature branches.
```
Comment (wuhuizuo) [inline pipelines/pingcap/tidb/latest/ghpr_build.groovy:103]:
```suggestion
            when { not { expression { REFS.base_ref ==~ /^feature[\/_]/ } } } // skip for feature branches.
```

File: pipelines/pingcap/tidb/release-7.1/ghpr_build.groovy [hunk 2]
```diff
@@ -104,6 +104,7 @@ pipeline {
             }
         }
         stage("Test plugin") {
+            when { not { expression { REFS.base_ref ==~ /^feature[\/_]/ } } // skip for feature branches.
```
Comment (wuhuizuo) [inline pipelines/pingcap/tidb/release-7.1/ghpr_build.groovy:107]:
```suggestion
            when { not { expression { REFS.base_ref ==~ /^feature[\/_]/ } } } // skip for feature branches.
```

File: pipelines/pingcap/tidb/release-7.5/ghpr_build.groovy [hunk 3]
```diff
@@ -99,6 +99,7 @@ pipeline {
             }
         }
         stage("Test plugin") {
+            when { not { expression { REFS.base_ref ==~ /^feature[\/_]/ } } // skip for feature branches.
```
Comment (wuhuizuo) [inline pipelines/pingcap/tidb/release-7.5/ghpr_build.groovy:102]:
```suggestion
            when { not { expression { REFS.base_ref ==~ /^feature[\/_]/ } } } // skip for feature branches.
```

File: pipelines/pingcap/tidb/release-7.6/ghpr_build.groovy [hunk 4]
```diff
@@ -99,6 +99,7 @@ pipeline {
             }
         }
         stage("Test plugin") {
+            when { not { expression { REFS.base_ref ==~ /^feature[\/_]/ } } // skip for feature branches.
```
Comment (wuhuizuo) [inline pipelines/pingcap/tidb/release-7.6/ghpr_build.groovy:102]:
```suggestion
            when { not { expression { REFS.base_ref ==~ /^feature[\/_]/ } } } // skip for feature branches.
```

**Comment by purelind** [comment]
replay test on master  https://do.pingcap.net/jenkins/blue/organizations/jenkins/pingcap%2Ftidb%2Fghpr_build/detail/ghpr_build/2707/pipeline

**Review by wuhuizuo — COMMENTED** [review]
> replay test on master  https://do.pingcap.net/jenkins/blue/organizations/jenkins/pingcap%2Ftidb%2Fghpr_build/detail/ghpr_build/2707/pipeline 

https://do.pingcap.net/jenkins/job/pingcap/job/tidb/job/ghpr_build/2708/

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**